<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentManagerSys</class>
 <widget class="QMainWindow" name="StudentManagerSys">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>StudentManagerSys</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 0);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_1">
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>651</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3"/>
    <widget class="QWidget" name="page_4"/>
    <widget class="QWidget" name="page_5"/>
    <widget class="QWidget" name="page_2"/>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>2</y>
      <width>666</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn1">
       <property name="minimumSize">
        <size>
         <width>128</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>128</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>学生信息</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn2">
       <property name="minimumSize">
        <size>
         <width>128</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>128</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>课程信息</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn3">
       <property name="minimumSize">
        <size>
         <width>128</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>128</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>成绩查询</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn4">
       <property name="minimumSize">
        <size>
         <width>128</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>128</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>成绩统计</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn5">
       <property name="minimumSize">
        <size>
         <width>128</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>128</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>用户修改</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
